keps/sig-storage/695-skip-permission-change/kep.yaml

```diff
@@ -25,7 +25,7 @@ stage: "alpha"
 milestone:
   alpha: "v1.18"
   beta: "v1.20"
-  stable: "v1.21"
+  stable: "v1.23"
```

Comment:
Please update stage above.

This also requires prod-readiness approval from PRR team (the above should force you to add a corresponding file prod-readiness dir:

https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-storage